app/models/user.rb

```diff
@@ -76,4 +77,12 @@ def allow_attended_to_event_notification?
   def allow_liked_event_notification?
     notification_timings.liked_event.present?
   end
+
+  def woman?
+    gender == "woman"
+  end
+
+  def only_woman_attend?(event)
+    event.only_woman != true || event.only_woman == true && woman?
```

Comment:
```suggestion
    !event.only_woman? || event.only_woman? && woman?
```
boolean型のカラムの場合、Railsがよしなに`カラム名?`というメソッドを定義してくれるので、それを使いましょう！